src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
```

Comment:
The extra variables aren't needed because the instances are used only one time. You can create new instances right in the line 8 `new Machine[] {new Truck(), ...}`